pkg/cli/cmd/radinit/aws.go

```diff
@@ -29,7 +29,10 @@ import (
 
 const (
 	selectAWSRegionPrompt                 = "Select the region you would like to deploy AWS resources to:"
+	selectAwsCredentialKindPrompt         = "Select a credential kind for the AWS credential:"
```

Comment:
We use `AWS` and `Aws` in two lines that follow each other.

Reply:
updated it